// A very simple protocol definition, consisting of only
// one message.
import "nanopb.proto";

message SecondMessage {
   option (nanopb_msgopt).msgid = 5678;
   required bool lucky_punch = 1;
}

